Documentation/Input/Pointers.md

```diff
@@ -245,6 +245,32 @@ The [PointerResultExample scene](https://github.com/microsoft/MixedRealityToolki
 
 <img src="../../Documentation/Images/Input/PointerResultExample.png" style="max-width:100%;">
 
+### Turn pointers on and off
+To turn pointers on and off, set the [`PointerBehavior`]() for a given pointer type via [`PointerUtils`](). See [How to turn pointers on and off in MRTK using PointerBehavior]() for more information.
+
+```csharp
+// Turn off all hand rays
+PointerUtils.SetHandRayPointerBehavior(PointerBehavior.Off);
+
+// Turn off hand rays for the right hand only
+PointerUtils.SetHandRayPointerBehavior(PointerBehavior.Off, Handedness.Right);
+
+// Turn off the gaze pointer
+PointerUtils.SetGazePointerBehavior(PointerBehavior.Off);
+
+// Set the behavior to match HoloLens 1
+// Note, if on HoloLens 2, must make the GGV pointer show up for articulated hands. See TurnPointersOnOffExample in the MRTK Examples package
```

Comment:
I really like that your example shows the typical setup for HL1 vs HL2 vs VR etc. 

Either 1) in this PR we should updated Pointers.md to talk about which pointers are typically valid on which platforms? or 2) file a documentation bug to do something similar later

Reply:
Thanks! I added a table

Reply:
Table looks great!

---
In reply to: [323377626](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5920#discussion_r323377626) [](ancestors = 323377626)